adobe/react-spectrum
# RSP-1861 [Docs][A11y] Manage focus when breadcrumb changes in link Popover

When a link within a popover changes the contents of the popover, focus will be lost to the document.body, because the element that had focus will be removed from the DOM and replaced with new content.  We should manage focus by explicitly setting focus to the current breadcrumb item when the content changes.


Closes [RSP-1861](https://jira.corp.adobe.com/browse/RSP-1861)

## ✅ Pull Request Checklist:

- [x] Included link to corresponding [Issue RSP-1861](https://jira.corp.adobe.com/browse/RSP-1861)

## 📝 Test Instructions:

1. yarn start:docs
2. Navigate to http://localhost:1234/react-aria/useListBox.html
3. Click on a link that opens a popover displaying information about the element type, like "AriaListBoxOptions".
4. Tab to another link to a React-Spectrum object type like "CollectionChildren".
5. Press Enter to activate the link, updating the contents of the Popover.
6. Focus will be set to the current breadcrumb, but since it is static content, no focus ring is displayed. Think of it like an in-page anchor link. 
7. Press Tab or Shift+Tab to navigate to the next or previous interactive control from the current focus in the tab navigation order.
8. The focus should move to the first focusable link after the Breadcrumbs in the Popover content or to the previous Breadcrumb Item from the current Breadcrumb

## 🧢 Your Project:

Accessibility
